packages/cli/src/util/output/create-output.ts

```diff
@@ -17,15 +19,17 @@ export interface LogOptions {
 export class Output {
   stream: WritableTTY;
   debugEnabled: boolean;
+  textOnly: boolean;
   private spinnerMessage: string;
   private _spinner: StopSpinner | null;
 
   constructor(
     stream: WritableTTY,
-    { debug: debugEnabled = false }: OutputOptions = {}
+    { debug: debugEnabled = false, textOnly = false }: OutputOptions = {}
```

Comment:
@TooTallNate
> I wonder if it would make more sense to have this be configurable via an env var or something more permanent. If the `--text-only` flag is needed to be specified for every command I think that could become annoying.

That makes sense.
What about use env var name as `TEXT_ONLY_MODE`?
If someone set `TEXT_ONLY_MODE=on` then default value of textOnly will be `process.env.TEXT_ONLY_MODE === 'on'`

Reply:
Like this

```suggestion
    { debug: debugEnabled = false, textOnly = process.env.TEXT_ONLY_MODE === 'on' }: OutputOptions = {}
```

Maybe `const IS_TEXT_ONLY_MODE = process.env.TEXT_ONLY_MODE === 'on';` then `{ debug: debugEnabled = false, textOnly = IS_TEXT_ONLY_MODE }: OutputOptions = {}` is better but...

Reply:
It could also be a global option for folks that prefer that configuration pattern

Reply:
[ef35bed](https://github.com/vercel/vercel/pull/8826/commits/ef35bed3ab3c096aa7d9eec336c2601b1131cf4a)

applied it

Is there any good way to guide this global option to folks that prefer this? l(ike doc or --help)